src/modules/consumer.js

```diff
@@ -1,17 +1,13 @@
 const parsers = require('./message-parsers');
 const utils = require('./utils');
-const assert = require('assert');
 
 class Consumer {
+
   constructor(connection) {
     this._connection = connection;
     this.channel = null;
   }
 
-  get connection() {
-    return this._connection;
-  }
-
   set connection(value) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide a getter matching this setter for 'connection'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3AS2376)